utils/polygonCutter.py

```diff
@@ -39,19 +40,18 @@ def CutPolygonFromArrayGDALds(self, polygon: Dict, tileNumber: int) -> np.ndarra
         # Resize the function cutting the array at the the size of the polygon
         array_chm = self.resizeMaskedArray(array_chm_cut)
 
-        plt.figure(5)
-        plt.imshow(mask)
-        plt.title('Binary Mask')
-
-        plt.figure(6)
-        plt.imshow(array_chm)
-        plt.title('CHM Filtered and cut for the desired building')
+        if self.flagPlots:
+            plt.figure(5)
+            plt.imshow(mask)
+            plt.title('Binary Mask')
 
+            plt.figure(6)
+            plt.imshow(array_chm)
+            plt.title('CHM Filtered and cut for the desired building')
 
         return array_chm
 
-    @staticmethod
-    def getCHMFromGDAL(tileNumber: int, tile_path: str = './data/{}_split/'):
+    def getCHMFromGDAL(self, tileNumber: int, tile_path: str = './data/{}_split/'):
```

Comment:
Since it's not a static method anymore, you can remove the tile_path argument and use self.tile_path instead on line 66 & 67. And thus remove the second argument in the function call line 24 too